skimage/transform/histogram_matching.py

```diff
@@ -49,11 +54,9 @@ def match_histograms(image, reference, multichannel=False):
     References
     ----------
     .. [1] http://paulbourke.net/miscellaneous/equalisation/
+    .. [2] https://www.researchgate.net/post/Histogram_matching_for_color_images
```

Comment:
I would prefer to see a paper rather than a researchgate thread.

Reply:
I have not found any paper about it yet, but see the following example
![hist-matching](https://user-images.githubusercontent.com/6035284/56847675-63542c80-68de-11e9-83df-66deed001ad2.png)

Reply:
I found a similar topic here - https://github.com/mapbox/rio-hist/issues/3

Reply:
If you had the example, you can remove this link. The example will speak by itself.